### PR TITLE
Adding Verification Levels to Test Support Matrix

### PR DESCRIPTION
**Pull request by kristyldatical**
Creating initial proposal

### CODE REVIEW

File: src/main/resources/liquibase/harness/generateChangelog/expectedChangeLog/edb-edb/createSequence.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <!-- Liquibase doesn't support 'dataType' attribute for mariadb, so it is removed comparing to basic changeLog-->
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/generateChangelog/expectedChangeLog/edb-edb/createSequence.xml:8]: need to change `mariadb` to `edb`

File: src/main/resources/liquibase/harness/snapshot/changelogs/cockroachdb/createSequence.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <!-- Liquibase doesn't support 'dataType' attribute for mariadb, so it is removed comparing to basic changeLog-->
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/snapshot/changelogs/cockroachdb/createSequence.xml:8]: need to change `mariadb` to `cockroachdb` if `datatype` isn't present for `cockroachdb` as well

File: src/main/resources/liquibase/harness/snapshot/changelogs/edb-edb/createSequence.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <!-- Liquibase doesn't support 'dataType' attribute for mariadb, so it is removed comparing to basic changeLog-->
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/snapshot/changelogs/edb-edb/createSequence.xml:8]: need to change `mariadb` to `edb` if `datatype` isn't present for `edb` as well

File: src/main/resources/liquibase/harness/snapshot/changelogs/h2/createSequence.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <!-- Liquibase doesn't support 'dataType' attribute for mariadb, so it is removed comparing to basic changeLog-->
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/snapshot/changelogs/h2/createSequence.xml:8]: same here need to change `mariadb` to `h2` if `datatype` isn't supported for `h2` as well

File: src/main/resources/liquibase/harness/generateChangelog/expectedChangeLog/cockroachdb/createFunction.sql [hunk 5]
```diff
@@ -0,0 +1,4 @@
+INVALID TEST
+
+Functions are not supported by Cockroach DB
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/generateChangelog/expectedChangeLog/cockroachdb/createFunction.sql:3]: cockroachdb v 22 supports user defined functions. Plz create separate ticket to unignore functions tests in all testcases

File: src/main/resources/liquibase/harness/generateChangelog/expectedChangeLog/firebird/createIndex.sql [hunk 6]
```diff
@@ -0,0 +1,3 @@
+INVALID TEST
+
+--Indexes are treated as unique constraints during generateChangelog command for Firebird
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/generateChangelog/expectedChangeLog/firebird/createIndex.sql:3]: is it expected behavior for liquibase?

**Review by KushnirykOleh — CHANGES_REQUESTED** [review]
left couple of comments that need to be addressed

File: src/main/resources/liquibase/harness/generateChangelog/expectedChangeLog/cockroachdb/createFunction.sql (hunk 5 above)
Comment (yodzhubeiskyi) [reply src/main/resources/liquibase/harness/generateChangelog/expectedChangeLog/cockroachdb/createFunction.sql:3]: As far as I can tell, cockroachdb supports functions only in v 22.2 and we are using 22.1 which still shows an error `You have attempted to use a feature that is not yet implemented.`

**Review by KushnirykOleh — APPROVED** [review]

**Pull request merged**